search/django/weedid/utils.py

```diff
@@ -45,3 +47,26 @@ def create_upload_entity(weedcoco_path, upload_id, upload_userid):
     upload_entity.save()
     upload_user.latest_upload = upload_entity
     upload_user.save()
+
+
+def remove_entity_local_record(user_id, upload_id):
+    upload_dir_record = os.path.join(UPLOAD_DIR, user_id, upload_id)
+    repository_dir_record = os.path.join(REPOSITORY_DIR, upload_id)
+    download_dir_record = os.path.join(DOWNLOAD_DIR, f"{upload_id}.zip")
+    for dirPath in [upload_dir_record, repository_dir_record, download_dir_record]:
```

Comment:
We should try to avoid camel case in Python. I should try to avoid camel case in Javascript. Or we should adopt a convention across both.